test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ComplexTypeInheritance/ComplexTypeInheritanceTests.cs

```diff
@@ -236,15 +236,17 @@ protected override void UpdateConfiguration(WebRouteConfiguration configuration)
         public async Task PatchContainingEntity(string modelMode)
         {
             string serviceRootUri = string.Format("{0}/{1}", BaseAddress, modelMode).ToLower();
-            string requestUri = serviceRootUri + "/Windows(3)";
+            string requestUri = serviceRootUri + "/Windows(1)";
 
             HttpRequestMessage request = new HttpRequestMessage(new HttpMethod("PATCH"), requestUri);
+
+            // We should be able to PATCH nested resource with delta object of the same CLR type.
             var content = @"
 {
     'CurrentShape':
     {
         '@odata.type':'#Microsoft.Test.E2E.AspNet.OData.ComplexTypeInheritance.Circle',  
-        'Radius':2,
```

Comment:
>2 [](start = 17, length = 1)

"2" can't work here?

Reply:
Windows(1)'s Radius has original value is already 2 (set by WindowsController()). Patching a different value of Radius=1 so that we can verify value changed on L271.

---
In reply to: [190751057](https://github.com/OData/WebApi/pull/1445#discussion_r190751057) [](ancestors = 190751057)